### ISSUE TITLE
Unknown error on Firefox 43 Windows 10 x64

### ISSUE THREAD

**Issue by flawyte** [webcomponents/webcomponentsjs#487]
I made a website using webcomponentjs but the page won't display anything except the background. When I open up the console there's a red line with only a colon `:` as error message, and it comes from the webcomponentsjs file. It happens with both the full and the lite versions.

Here's my index.html file:

``` html

<!DOCTYPE html>
<html lang="en">
<head>
  <title>Make a Wallpaper</title>
  <script src="/bower_components/webcomponentsjs/webcomponents-lite.min.js"></script>
  <script src="index.js"></script>
  <meta charset="utf-8">
  <link rel="stylesheet" href="index.css">
  <link rel="import" href="/bower_components/google-analytics/google-analytics.html">
  <link rel="import" href="/components/facebook-buttons.html">
  <link rel="import" href="/components/maw-error.html">
  <link rel="import" href="/components/maw-form.html">
  <link rel="import" href="/components/maw-spinner.html">
  <link rel="import" href="/components/maw-success.html">
  <link rel="import" href="/components/twitter-share-button.html">
</head>
<body unresolved>
  <google-analytics code="UA-XXXXXXXX-4"></google-analytics>
  <div id="main">
    <div id="title">
      <h1>MaW</h1>
      <span>Make a Wallpaper</span>
    </div>
    <maw-form></maw-form>
    <maw-error></maw-error>
    <maw-spinner></maw-spinner>
    <maw-success></maw-success>
    <footer>
      <facebook-buttons></facebook-buttons>
      <twitter-share-button></twitter-share-button>
    </footer>
  </div>
</body>
</html>
```

It happens on Firefox 43 running on Windows 10 x64. However on another computer running Windows 10 x64 and Firefox 44 it works as expected... Any ideas? Maybe a line to comment in the source file?

**Comment by azakus** [comment]
Do you have `dom.webcomponents.enabled` set on the failing one [mdn link](https://developer.mozilla.org/en-US/docs/Web/Web_Components#Enabling_Web_Components_in_Firefox)? If so, turn it off.

**Comment by flawyte** [comment]
No it's disabled.

**Comment by flawyte** [comment]
Okay I found the source of the error. It comes from elements that need to execute a script to work, for example if the element wraps a facebook like button, a twitter button or a google analytics script.

For example, [this element](https://github.com/cesarwbr/google-analytics-element/blob/gh-pages/google-analytics.html) make the webcomponents script to fail, as well as custom elements I've made to wrap facebook and twitter share buttons. Except that in my components, I've placed the JS code into the `attached()` function but anyway the result is the same, webcomponentsjs throws a `:` error in some versions of Firefox.

Now you just have to find why it does this.
